.github/workflows/cygwin.yml

```diff
@@ -1,10 +1,7 @@
 name: "Cygwin"
 
-on:
-  push:
-    branches: [ latest ]
-  pull_request:
-    branches: [ latest ]
+on: 
+  workflow_dispatch
```

Comment:
This file shouldn't be changed - restore it to the original version now that your testing is complete.

Reply:
Oh that was not intended to happen, must have been merged from a different branch.